src/Form/FormatConfigurationFormPdf.php

```diff
@@ -0,0 +1,150 @@
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides shared configuration form for all printable formats.
+ */
+class FormatConfigurationFormPdf extends FormBase {
+
+  /**
+   * The printable entity manager.
+   *
+   * @var \Drupal\printable\PrintableEntityManagerInterface
+   */
+  protected $printableEntityManager;
+
+  /**
+   * Constructs a new form object.
+   *
+   * @param \Drupal\printable\PrintableEntityManagerInterface $printable_entity_manager
+   *   The printable entity manager.
+   */
+  public function __construct(PrintableEntityManagerInterface $printable_entity_manager) {
+    $this->printableEntityManager = $printable_entity_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('printable.entity_manager')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'printable_configuration_pdf';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state, $printable_format = NULL) {
+    $form['settings'] = array(
+      '#type' => 'fieldset',
+      '#title' => $this->t('PDF options'),
+    );
+    $form['settings']['print_pdf_pdf_tool'] = array(
+      '#type' => 'radios',
+      '#title' => $this->t('PDF generation tool'),
+      '#options' => array('mPDF' => 'mPDF', 'wkhtmltopdf' => 'wkhtmltopdf', 'TCPDF' => 'TCPDF'),
+      '#default_value' => (string)$this->config('printable.settings')->get('pdf_tool'),
+      '#description' => $this->t('This option selects the PDF generation tool being used by this module to create the PDF version.'),
+    );
+    $form['settings']['print_pdf_content_disposition'] = array(
+      '#type' => 'checkbox',
+      '#title' => $this->t('Save the pdf'),
+      '#description' => $this->t('Save the pdf instead of showing inline'),
+      '#default_value' => $this->config('printable.settings')->get('save_pdf'),
+    );
+    $form['settings']['print_pdf_paper_size'] = array(
+      '#type' => 'select',
+      '#title' => $this->t('Paper size'),
+      '#options' => array(),
+      '#default_value' => (string)$this->config('printable.settings')->get('paper_size'),
+      '#description' => $this->t('Choose the paper size of the generated PDF.'),
+    );
+    $paper_sizes = array(
+      'A0',
+      'A1',
+      'A2',
+      'A3',
+      'A4',
+      'A5',
+      'A6',
+      'A7',
+      'A8',
+      'A9',
+      'B0',
+      'B1',
+      'B10',
+      'B2',
+      'B3',
+      'B4',
+      'B5',
+      'B6',
+      'B7',
+      'B8',
+      'B9',
+      'C5E',
+      'Comm10E',
+      'DLE',
+      'Executive',
+      'Folio',
+      'Ledger',
+      'Legal',
+      'Letter',
+      'Tabloid',
+    );
+    foreach ($paper_sizes as $entity_type ) {
+      $form['settings']['print_pdf_paper_size']['#options'][$entity_type] = $entity_type;
+    }
```

Comment:
$entity_type is a very confusing name here. Just use the singular of $paper_sizes.